ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -3117,7 +3117,7 @@ tests:
       OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
     test:
     - chain: cucushift-sdn-migration-ovn
```

Comment:
@zhaozhanqi Could you try to[ test chain: cucushift-sdn-live-migration-ovn](url) in 4.16?

Reply:
Update live migration in another job `vsphere-ipi-sdn-live-migration-ovn-f28`